src/modules/dbus/dbusmodule.cpp

```diff
@@ -631,6 +672,8 @@ class Controller1 : public ObjectVTable<Controller1> {
                                "s", "");
     FCITX_OBJECT_VTABLE_METHOD(currentInputMethodGroup,
                                "CurrentInputMethodGroup", "", "s");
+    FCITX_OBJECT_VTABLE_METHOD(getIMInfoListByGroupName,
+                               "GetIMInfoListByGroupName", "s", "a{sv}");
```

Comment:
I expect the return value type to be a(sssssssba{sv})

which means it is an array of InputMethodEntry.

7 "s" & 1 "b" means the existing field, uniqueName etc.. (see AvailableInputMethods, that one is missing "addon" though)
a{sv} means the field reserved for the future, which can only be empty at this point.